recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e188841f87379bf01105fd6b5d4649a8183d6de2be2c8b7677ea2b7d750ce7c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy ==1.19.1
+    - pandas ==1.1.1
+    - ipywidgets ==7.5.1
+    - pydash ==4.8.0
+    - scikit-learn ==0.23.2
+
+test:
+  imports:
+    - ipychart
+
+about:
+  home: https://github.com/nicohlr/ipychart
+  license: MIT
+  license_family: MIT
+  # Reminder : check if the source distribution includes the license file in future releases.
+  # PR which add LICENSE to source distribution: https://github.com/nicohlr/ipychart/pull/1
+  license_file: LICENSE.txt
+  summary: 'A Jupyter - Chart.js bridge'
+
+  description: | 
+    ipychart is an ipywidget which allows to create dynamic, refined 
+    and customizable charts within the famous Jupyter Notebook environment.
+  doc_url: https://nicohlr.gitlab.io/ipychart/
+  dev_url: https://github.com/nicohlr/ipychart
```

Comment:
You need the following section:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L78-L83